examples/typescript/actors/http-hello-world/wit/deps/io/poll.wit

```diff
@@ -3,7 +3,7 @@ package wasi:io@0.2.0-rc-2023-11-10;
 /// A poll API intended to let users wait for I/O events on multiple handles
 /// at once.
 interface poll {
-    /// `pollable` represents a single I/O event which may be ready, or not.
+    /// `pollable` epresents a single I/O event which may be ready, or not.
```

Comment:
![tenor-112006303](https://github.com/wasmCloud/wasmCloud/assets/3869136/febc44e8-936a-4dfa-82a4-5c716334e9a3)

Reply:
😢 I just copied it over from the Rust folder, gonna merge for now and just deal with it since we should update to 0.2 anyways soon

Reply:
I know, it's not the first time I've teased you about typos in wit that you didn't write. Still cracks me up every time